libs/ui-lib/lib/common/configurations.ts

```diff
@@ -0,0 +1,2 @@
+export const MAX_FILE_SIZE_BASELINE = 1; //1 Mb
+export const MAX_FILE_SIZE_OFFSET_FACTOR = MAX_FILE_SIZE_BASELINE * 2 * 1048576; //2 Mib
```

Comment:
This makes the MAX_FILE_SIZE_OFFSET_FACTOR the value used for the validations. That was not what I meant.
The idea was to have 2 constants, the value used for the validations would be obtained by calling a function (let's call it `getMaxFileSizeValidationValue`) that performs the calculation you did in line 2.
```suggestion
export const MAX_FILE_SIZE_OFFSET_FACTOR = 2 * 1024**2; //2 MiB
```

Reply:
Ok, but your calculation is not correct. We need to validate the size in bytes

Reply:
Let's do the following in order to attempt to make the code more readable:
Rename MAX_FILE_SIZE_BASELINE -> MAX_FILE_SIZE_BYTES, set it to 1024**2 (`1024**2=1048576` or 1MiB)
Set MAX_FILE_SIZE_OFFSET_FACTOR = 2

We use a library called filesize.js for converting units into SI and IEC standards and also into a user friendly string (see `libs/ui-lib/lib/cim/components/Hypershift/DetailsPage/NodePoolsTable.tsx:177`). PTAL and try to reuse it.